managed/src/main/java/com/yugabyte/yw/commissioner/tasks/UniverseTaskBase.java

```diff
@@ -999,6 +999,20 @@ public SubTaskGroup createTableBackupTask(BackupTableParams taskParams, Backup b
     return subTaskGroup;
   }
 
+   public SubTaskGroup createDeleteBackupTasks(List<Backup> backups, UUID customerUUID) {
```

Comment:
Perfect! 
I was looking at previous commit and was going to comment:
"Have not looked deeply but can we just submit DeleteBackup task instead of reinventing the wheel?
Or at least share code with that task. But I prefer former. "